pkgs/development/coq-modules/coq-extensible-records/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl, coq }:
+
+stdenv.mkDerivation {
+  name = "coq${coq.coq-version}-coq-extensible-records-1.2.0";
+
+  src = fetchurl {
+    url = https://github.com/gmalecha/coq-extensible-records/archive/1.2.0.tar.gz;
+    sha256 = "031yiwsy7zkm9nr0nk5cj3h069bjkymc6z5fx84q4l1n9xqp1ysq";
+  };
+
+  buildInputs = [ coq ];
+  enableParallelBuilding = false;
+
+  installPhase = ''
+    make Makefile.coq
```

Comment:
Is this needed?

Reply:
It's an optimization (see: https://github.com/gmalecha/coq-extensible-records/blob/master/Makefile). Not sure whether every user would want to have that optimization.

Reply:
Yeah, `all` points to some weird targets like `example`. I changed it to `make` instead, it does not matter much and we might as well.

Reply:
Still, I don’t get why `make` should be called again at the beginning of the `installPhase`.

Reply:
Oh, that's my mistake, I guess I didn't realize `make` was already called by then. I removed that line and tried again, it should still work now and do just the necessary!